README.md

```diff
@@ -182,7 +182,26 @@ OSTk Physics uses input data from various sources to determine the state of the
 
 None of these files are shipped with the source code of this library. OSTk Physics has the capability to fetch the required files at runtime if they are not present or if they are outdated. This is done using file Manager classes (see any file named `Manager.hpp`). Data for any use-case is queried through the Manager class rather than directly, which allows the Manager to handle file loading and fetching.
 
-The following table shows the availabe data source settings:
+If you would like to seed the ostk data generation so that you already have an initial copy of all the data files (to avoid re-fetching at runtime), then we recommend adding these lines of code to the dockerfile you use to build your production image where OSTk is installed.
```

Comment:
```suggestion
If you would like to seed the ostk data generation so that you already have an initial copy of all the data files (to avoid re-fetching at runtime), then we recommend running the following commands to set up your environment.
```

nit: not everyone is running OSTk in Docker